ltp/data.py

```diff
@@ -124,16 +162,51 @@ def read_runtest(suite_name: str, content: str) -> Suite:
         if len(parts) >= 3:
             test_args = parts[2:]
 
-        test = Test(test_name, test_cmd, test_args)
+        parallelizable = True
+
+        if not metadata_tests:
+            # no metadata no party
+            parallelizable = False
+        else:
+            test_params = None
+            for mtest_name, mtest_params in metadata_tests.items():
```

Comment:
Potentially there could be 10K+ tests in the metadata and 2-3K lines in a runtest file. We probably want a dict of mtest_name -> mtest_params.